changelog/unreleased/bugfix-turned-off-file-extensions-had-no-effect

```diff
@@ -0,0 +1,6 @@
+Bugfix: Turned off file extensions had no effect
```

Comment:
The title is a bit over-generalized 😅

Reply:
Eino but the 80 char limit is fucking around with me

Reply:
```suggestion
Bugfix: Turned off file extensions not always respected
```
maybe?